frontend/orders-pane.js

```diff
@@ -789,7 +793,12 @@ class OrdersPane extends HTMLElement {
 			};
 			shadow.getElementById("final_action_details").innerHTML = final_act_desc[shadow.getElementById("final_action").value];
 		});
-		if (g_data.date % 7 != 0) for (let e of shadow.querySelectorAll("#score_switches input")) e.disabled = true;
+		if (g_data.date % 7 == 0) {
+			shadow.querySelector("#tab_game").className = "tab_game alert";
```

Comment:
Perhaps a little paranoid but in terms of future-proofing for errors, I think appending " alert" to the existing className is safer than overwriting it completely.  This way if you want to use this pattern repeatedly you don't have to worry about overwriting changes.

Reply:
Done. Turns out there is an API for this.